lib/init.coffee

```diff
@@ -7,6 +7,9 @@ module.exports =
       type: 'string'
       default: 'clojure-x.x.x.jar'
 
+  activate: ->
+    require("atom-package-deps").install("linter-clojure")
```

Comment:
Also the name is now optional with `atom-package-deps` v3, see [changelog](https://github.com/steelbrain/package-deps/blob/master/CHANGELOG.md#300)